### DISCUSSION TITLE
Does the Probe manifest support ability to send host headers?

### DISCUSSION THREAD

**Discussion by kherrmann3** [prometheus-operator/kube-prometheus#1805] [Q&A] [answered]
I have a service in kubernetes that I'm trying to monitor, from the perspective of other services within the cluster, but accessing it over its service name doesn't send the proper host header. Is it possible to configure that using the "Kind: Probe" object enabled with kube-prometheus? 

The only examples I've seen have been very simple URL checks.

Thank you!

**Comment by simonpasquier** [comment, marked as answer]
I believe that it's something you want to setup at the [blackbox exporter level](https://github.com/prometheus/blackbox_exporter/blob/master/CONFIGURATION.md#http_probe).

**Reply by kherrmann3** [reply]
Thanks...that appears to be consistent with what I discovered while researching after posting. Thank you.
